### PR TITLE
docs/execution_environments.md: add a link to EE getting started guide

### LINKED ISSUE #311: AWX docs contain a reference to EE getting started
https://github.com/ansible/awx/pull/14263

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY

Fixes https://github.com/ansible-community/community-team/issues/311

This replaces the EE links with a link to recently created Getting-started with EE guide

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

```
devel
```

### CODE REVIEW

File: docs/execution_environments.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 All jobs use container isolation for environment consistency and security.
 Compliant images are referred to as Execution Environments (EE)s.
-For more information, see [ansible-runner EE docs](https://ansible-runner.readthedocs.io/en/latest/execution_environments.html)
-for information on how container isolation works, and see [ansible-builder docs](https://ansible-builder.readthedocs.io/en/latest/index.html)
-for instructions on how to build them.
+
+For more information about the EE technology as well as how to build and test EEs, see the
+[Getting started with Execution Environments guide](https://docs.ansible.com/ansible/devel/getting_started_ee/index.html).
```
Comment (samccann) [inline docs/execution_environments.md]: We should also keep a link to the actual builder docs here imo.

Comment (samccann) [reply docs/execution_environments.md]: as in https://ansible.readthedocs.io/projects/builder/en/latest/

Comment (tvo318) [reply docs/execution_environments.md]: I agree!

**Review by tvo318 — COMMENTED** [review]
Looks good with the addition of @samccann's feedback.

**Comment by Andersson007** [comment]
> We should also keep a link to the actual builder docs here imo.

SGTM, done, thanks PTAL

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Not my space/place to merge, but LGTM

**Review by fosterseth — APPROVED** [review]

**Comment by Andersson007** [comment]
thanks everyone!

**Pull request merged**